clang/lib/Driver/OffloadBundler.cpp

```diff
@@ -1365,6 +1367,20 @@ Error OffloadBundler::BundleFiles() {
   return Error::success();
 }
 
+bool CheckTripleIsExcluded(
+    StringRef Triple, const std::vector<std::string> &ExcludedTargetNames) {
+  // NOTE: "-sycldevice" Triple component has been deprecated.
+  // However, it still can be met in libraries that have been compiled before
+  // deprecation. For example, here Triple might be the following:
+  //  sycl-fpga_aoco-intel-unknown-sycldevice
+  //
+  // The workaround is to strip this Triple component if it is present.
+  Triple.consume_back("-sycldevice");
+  auto it =
```

Comment:
We can use: 
return ExcludedTargetNames.count(Triple);

Reply:
`std::vector` lacks this functionality. And it isn't expected to see here a long list of excluded targets to win some performance using `std::unordered_set`.